docs/source/datamodules.rst

```diff
@@ -1,3 +1,7 @@
+.. testsetup:: *
+
+    from pytorch_lightning.core.datamodule import LightningDataModule
```

Comment:
let's move this to the global import in conf.py

Reply:
sure.